scripts/generators/spirv_validation_generator.py

```diff
@@ -187,25 +187,33 @@ def generate(self):
         #
         # Build the struct with all the requirments for the spirv capabilities
         out.append('// clang-format off\n')
-        out.append('static const std::unordered_multimap<uint32_t, RequiredSpirvInfo> spirvCapabilities = {\n')
+        out.append('const std::unordered_multimap<uint32_t, RequiredSpirvInfo>& GetSpirvCapabilites() {\n')
+        out.append('    static const std::unordered_multimap<uint32_t, RequiredSpirvInfo> spirv_capabilities = {')
         for spirv in [x for x in self.vk.spirv if x.capability]:
             for enable in [x for x in spirv.enable if x.struct is None or x.struct not in self.promotedFeatures]:
                 if spirv.name not in self.capabilityList:
-                    out.append('    // Not found in current SPIR-V Headers\n    //')
-                out.append(f'    {{spv::Capability{spirv.name}, {self.createMapValue(spirv.name, enable, False)}}},\n')
+                    out.append('\n    // Not found in current SPIR-V Headers\n    // ')
+                else:
+                    out.append('\n    ')
+                out.append(f'{{spv::Capability{spirv.name}, {self.createMapValue(spirv.name, enable, False)}}},')
+        out.append('};\n')
+        out.append('    return spirv_capabilities;')
         out.append('};\n')
         out.append('// clang-format on\n')
         out.append('\n')
 
         #
         # Build the struct with all the requirments for the spirv extensions
+        out.append('const std::unordered_multimap<std::string_view, RequiredSpirvInfo>& GetSpirvExtensions() {\n')
         out.append('// clang-format off\n')
-        out.append('static const std::unordered_multimap<std::string_view, RequiredSpirvInfo> spirvExtensions = {\n')
+        out.append('    static const std::unordered_multimap<std::string_view, RequiredSpirvInfo> spirv_extensions = {')
         for spirv in [x for x in self.vk.spirv if x.extension]:
             for enable in spirv.enable:
-                out.append(f'    {{"{spirv.name}", {self.createMapValue(spirv.name, enable, True)}}},\n')
+                out.append(f'\n    {{"{spirv.name}", {self.createMapValue(spirv.name, enable, True)}}},')
         out.append('};\n')
```

Comment:
```suggestion
        out.append('\n};\n')
        out.append('    return spirv_extensions;\n')
        out.append('}\n')
        out.append('// clang-format on\n')        
        out.append('\n')
```

maybe better to just go

`out.append(''' .... ''')` 

here